pegasus/sites.v3/code.org/public/css/minecraft.css

```diff
@@ -10,8 +10,12 @@
 }
 
 .mc-new-tag {
-  float: right;
-  margin-bottom: 30px;
+  position: absolute;
+  top: 10px;
+  right: 10px;
+  width: 98px;
+  image-rendering:pixelated;
+  -ms-interpolation-mode:nearest-neighbor;
```

Comment:
hah

Reply:
nit: spaces after :s

Reply:
also, does that form of interpolation actually make sense given the source images below?  they seem full of anti-aliased diagonals rather than straight up pixel art.

Reply:
This is just for the pixelated "New" tag:

![image](https://user-images.githubusercontent.com/413693/32505161-53109e10-c396-11e7-9479-2c0fa4afd1f5.png)